tools/proteomiqon_peptidedb/proteomiqon_peptidedb.xml

```diff
@@ -0,0 +1,206 @@
+<tool id="proteomiqon_peptidedb" name="ProteomIQon PeptideDB" version="@VERSION@" python_template_version="20.05">
+    <description>
+		creates a peptide database in the SQLite format.
+	</description>
+    <macros>
+		<token name="@VERSION@">0.0.5</token>
+	</macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">proteomiqon-peptidedb</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        #if $outputParamfile:
+            cat '$paramfile' >> '$out_paramfile' &&
+        #end if
+        proteomiqon-peptidedb -i '$fasta' -p '$paramfile' -o ./ &&
+        cp './${Name}.db' '$out_db'
+            ]]>
+    </command>
+    <configfiles>
+        <configfile name="paramfile">
+        <![CDATA[
+        {
+            "Name": "${Name}",
+            "ParseProteinIDRegexPattern": "${ParseProteinIDRegexPattern}",
+            "Protease": { "Case": "${Protease}" },
+            "MinMissedCleavages": ${MinMissedCleavages},
+            "MaxMissedCleavages": ${MaxMissedCleavages},
+            "MaxMass": ${MaxMass},
+            "MinPepLength": ${MinPepLength},
+            "MaxPepLength": ${MaxPepLength},
+            "IsotopicMod": [
+            #set isotopicModList = $list($IsotopicMod)
+            #for $mod in $range($len($isotopicModList))
+                #if $mod != $len($isotopicModList) -1
+                {"Case":"${isotopicModList[$mod]}"},
+                #else
+                {"Case":"${isotopicModList[$mod]}"}
+                #end if
+            #end for
+            ],
+            "MassMode": {
+                "Case": "${MassMode}"
+            },
+            "FixedMods": [
+            #set fixedModList = $list($FixedMods)
+            #for $mod in $range($len($fixedModList))
+                #if $mod != $len($fixedModList) -1
+                {"Case":"${fixedModList[$mod]}"},
+                #else
+                {"Case":"${fixedModList[$mod]}"}
+                #end if
+            #end for
+            ],
+            "VariableMods": [
+            #set variableModList = $list($VariableMods)
+            #for $mod in $range($len($variableModList))
+                #if $mod != $len($variableModList) -1
+                {"Case":"${variableModList[$mod]}"},
+                #else
+                {"Case":"${variableModList[$mod]}"}
+                #end if
+            #end for
+            ],
+            "VarModThreshold": ${VarModThreshold}
+            }
+            ]]>
+        </configfile>
+    </configfiles>
+    <inputs>
+        <param name="fasta" type="data" format="fasta" label="Fasta file" help="Please specify your protein sequences in the .fasta format. If you want to search for contaminants you have to include the sequences in your file. You do not have to supply decoy sequences, those are generated for you."/>
+        <param name="Name" type="text" value="" label="Database name" />
```

Comment:
is that really needed? can we not use a fixed one? the user will not see it? Something like galaxy.db?

Reply:
Indeed, will change with next commits!